src/navigate-pop/using-workspaces.md

```diff
@@ -0,0 +1,30 @@
+# Workspaces
+
+Manage multiple workflows and projects with Workspaces. Show and access Workspaces by clicking on the icon with multiple windows in the dock, clicking `Workspaces` in the top left corner of your screen, or by pressing `SUPER` + `D`.
+
+![Show Workspaces](/images/workspaces/show-workspaces.png)
+
+The open windows for a given Workspace will fan across the screen when that workspace is selected.
+
+![Show Windows & Workspaces](/images/workspaces/show-windows-workspaces.png)
+
+Click and drag applications between Workspaces as needed.
+
+![Drag Window to Workspace](/images/workspaces/drag-window-workspace.png)
+
+## Navigate Workspaces with Keyboard Shortcuts
+
+Pop!\_OS includes these default key binds allowing users to quickly navigate between windows and workspaces.
```

Comment:
`key bindings` might be more grammatically correct than `key binds`.

Reply:
After review, this sentence seems redundant since the section is self-explanatory.